test/Solana.SvmCpiEventsClient.Integration.test.ts

```diff
@@ -147,4 +237,16 @@ describe("SvmCpiEventsClient (integration)", () => {
     expect(events).to.have.lengthOf(1);
     expect(events[0].data.inputAmount).to.equal(secondDeposit.inputAmount);
   });
+
+  it.only("creates and reads a single request slow fill event", async () => {
+    await sendRequestSlowFill();
+
+    const [requestSlowFillEvent] = await client.queryEvents("RequestedSlowFill");
+
+    const { data } = requestSlowFillEvent as { data: SvmSpokeClient.RequestedSlowFill };
+
+    expect(data.depositor).to.equal(signer.address.toString());
+    expect(data.recipient).to.equal(signer.address.toString());
+    expect(data.inputToken).to.equal(mint.address.toString());
```

Comment:
Is it work constructing `sendRequestSlowFill()` here such that it accepts a RelayData input? Then we can compare the RelayData hashes on the output side to verify that the request was submitted correctly. This would also be convenient as a utility function down in the relayer, because we currently don't have any slow fill request support for SVM.

Reply:
I added the generic `deposit`, `requestSlowFill`, and `createFill` functions here:
https://github.com/across-protocol/sdk/blob/02128c254326f137826c10654a31d60bde5548a4/test/utils/svm/utils.ts#L264-L339

They take SVM-typed inputs to keep things flexible.

For the event tests, I made simpler wrappers that also return the relayData object, so it’s easy to check that the events are logging correctly:
https://github.com/across-protocol/sdk/blob/02128c254326f137826c10654a31d60bde5548a4/test/Solana.SvmCpiEventsClient.Integration.test.ts#L70-L205